docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'

```

Comment:
Move docker discussion to Issues, not in this PR. 

Keep PRs narrow and focused.

Reply:
For those of us that need a quick tutorial on [docker](https://hackernoon.com/docker-tutorial-getting-started-with-python-redis-and-nginx-81a9d740d091) (...me! that's me. I need a tutorial)

Reply:
Sure, that's fine. That was an artifact of me trying to wrangle my dev environment to operate in a Linux container. :-P I've switched over to development on Windows from OS X and I was trying to find a solution that would work for me but could also help later when it came time to deploy.